core/src/main/java/dev/streamx/cli/ExecutionExceptionHandler.java

```diff
@@ -9,7 +9,7 @@
 import picocli.CommandLine.ParseResult;
 
 @ApplicationScoped
-public class ExceptionHandler implements IExecutionExceptionHandler {
+public class ExecutionExceptionHandler implements IExecutionExceptionHandler {
```

Comment:
Added ParameterExceptionHandler, so make this existing handler's name more meaningful